Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -188,7 +188,8 @@ const argumINTEnd = csr3
 const argumINTDsp = csr4
 
 macro ipintEntry()
-    checkStackOverflow(ws0, argumINTEnd)
+    const argumINTEndAsScratch = argumINTEnd
+    checkStackOverflow(ws0, argumINTEndAsScratch)
```

Comment:
Why isn’t this same fix needed in the other file?

Reply:
This isn't a fix, but just a way of documenting that we're re-using the `argumINTEnd` register as a scratch here, and not doing anything with the `argumINTEnd` value as the name previously suggested.  We can do without it, but I thought it added value here.  So, I only added it here.  The 32-bit file was already using a `Tmp` register, and the name implies it's a scratch.  We can't use `argumINTTmp` here because on 64-bit, it aliases with another register which is still live up to that point.